source/preprocessor.tex

```diff
@@ -767,8 +778,8 @@
 \pnum
 \indextext{__VA_OPT__@\mname{VA_OPT}}%
 The identifier \mname{VA_OPT}
-shall always occur as part of the token sequence
-\mname{VA_OPT}\tcode{(}\placeholder{content}\tcode{)},
+shall always occur as part of the preprocessing token sequence
+\mname{VA_OPT}\tcode{(}\placeholder{cont\-ent}\tcode{)},
```

Comment:
We should describe `__VA_OPT__(`*content*`)` in the grammar and give it a production name like we do for *defined-macro-expression* and *has-include-expression*.

Reply:
Let's leave this one for after the motions.

Reply:
Filed issue https://github.com/cplusplus/draft/issues/2185 to track this.